kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/VirtualKafkaClusterReconciler.java

```diff
@@ -141,6 +158,18 @@ public UpdateControl<VirtualKafkaCluster> reconcile(VirtualKafkaCluster cluster,
                     && unresolvedIngresses.isEmpty()
                     && unresolvedFilters.isEmpty()
                     && mismatchedIngressProxyRef.isEmpty()) {
+
+                var ingresses = cluster.getSpec().getIngressRefs()
+                        .stream()
+                        .filter(ingressRef -> existingKubernetesServices.containsKey(ingressRef.getName()))
+                        .map(ingressRef -> {
+                            var kubenetesService = existingKubernetesServices.get(ingressRef.getName());
+                            var builder = new IngressesBuilder();
+                            builder.withName(ingressRef.getName());
+                            builder.withBootstrap(kubenetesService.getMetadata().getName() + "." +  kubenetesService.getMetadata().getNamespace() + ".svc.cluster.local");
```

Comment:
Not all kube clusters use the `.cluster.local` domain, so this needs to be either configurable, or (better) discovered using operator startup.

Reply:
I think this is worth a separate issue #2056 as there may be several options.  I think this PR is okay to assume the default.